apps/developer-hub/src/components/Root/theme.css

```diff
@@ -334,3 +334,16 @@
   );
   --color-fd-ring: light-dark(var(--color-violet-600), var(--color-violet-400));
 }
+
+/* Global typography: tighten line spacing in docs content */
+:where(.prose) {
+  line-height: 1.5;
+}
+
+:where(.prose) :where(p, li) {
+  line-height: 1.5;
+}
+
+:where(.prose) :where(h1, h2, h3, h4, h5, h6) {
+  line-height: 1.2;
+}
```

Comment:
I very strongly recommend against setting global styles; instead we should be scoping styles to components.

Reply:
@alexcambose 
Can you please fix this in your next PR?